src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.cs

```diff
@@ -1865,19 +1865,27 @@ public async Task PostAsync_CallMethod_EmptyContent(Configuration.Http.RemoteSer
             }
         }
 
+        public static IEnumerable<object[]> ExpectContinueVersion()
+        {
+            var versions = new string[] {"1.0", "1.1", "2.0"};
+            var expectContinue = new bool?[] {true, false, null};
+            return
+                from expect in expectContinue
+                from version in versions
+                select new object[] {expect, version};
+        }
+
         [OuterLoop("Uses external server")]
         [Theory]
-        [InlineData(false, "1.0")]
-        [InlineData(true, "1.0")]
-        [InlineData(null, "1.0")]
-        [InlineData(false, "1.1")]
-        [InlineData(true, "1.1")]
-        [InlineData(null, "1.1")]
-        [InlineData(false, "2.0")]
-        [InlineData(true, "2.0")]
-        [InlineData(null, "2.0")]
+        [MemberData(nameof(ExpectContinueVersion))]
         public async Task PostAsync_ExpectContinue_Success(bool? expectContinue, string version)
         {
+            // Sync API supported only up to HTTP/1.1
+            if (!TestAsync && version == "2.0")
```

Comment:
Maybe:
```suggestion
            if (!TestAsync && !version.StartsWith("1."))
```
?

Reply:
Changed the type of `version` from `string` to `Version` and used `version.Major >= 2` as is in other tests.